x-pack/docs/en/rest-api/security/get-builtin-privileges.asciidoc

```diff
@@ -66,6 +66,7 @@ A successful call returns an object with "cluster" and "index" fields.
     "delegate_pki",
     "manage",
     "manage_api_key",
+    "manage_autoscaling",
```

Comment:
I think this leaks a bit of docs on autoscaling into release builds. I think this is fair (completely hiding it is probably too much trouble), but maybe we need a short alternative autoscaling docs description explaining this (in a follow-up)?